rules/apps.libsonnet

```diff
@@ -216,7 +238,7 @@
                     "replicaset", "$1", "owner_name", "(.*)"
                   ) * on(replicaset, namespace) group_left(owner_name) topk by(replicaset, namespace) (
                     1, max by (replicaset, namespace, owner_name) (
-                      kube_replicaset_owner{%(kubeStateMetricsSelector)s}
+                      kube_replicaset_owner{%(kubeStateMetricsSelector)s, owner_kind="Deployment"}
```

Comment:
could you also please ensure all workload rules use the cluster label for every aggregation? looks like it's missing in some places for existing rules